editor/src/clj/editor/texture_set.clj

```diff
@@ -265,9 +265,9 @@
 
 (defn- animation-frame->vertex-pos-uv
   [animation-frame world-transform]
-  (let [^double width (:width animation-frame)
-        ^double height (:height animation-frame)
-        frame-vertex-data (frame-vertex-data animation-frame width height)
+  (let [size [(:width animation-frame)
+              (:height animation-frame)]
+        frame-vertex-data (frame-vertex-data animation-frame size :pivot-center)
```

Comment:
I changed the argument list to `frame-vertex-data` in order to support pivot transformations, but apparently I missed this call.